content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -78,9 +77,9 @@ read that resource will fail until it is deleted or a valid decryption key is pr
 Name | Encryption | Strength | Speed | Key Length | Other Considerations
 -----|------------|----------|-------|------------|---------------------
 `identity` | None | N/A | N/A | N/A | Resources written as-is without encryption. When set as the first provider, the resource will be decrypted as new values are written.
-`aescbc` | AES-CBC with PKCS#7 padding | Strongest | Fast | 32-byte | The recommended choice for encryption at rest but may be slightly slower than `secretbox`.
```

Comment:
Why the reordering? I don't think it's strongest-first - `identity` is not an effective confidentiality mechanism!

Reply:
+1, why the reordering? The strongest seems to be the last in the table, `kms`, which.....

Reply:
That's a good point. Do you think its better separated into two tables? "Recommended" which contains (secretbox, AES-GCM, and KMS), and "Others" which contains (CBC, identity)?

Reply:
I would change “Strongest” to “Weak”, update the Other Considerations column as well, and not change the order. That'd be all that I think we need to do.

Reply:
kms also use aes-cbc mode, only difference is that the encryption key is managed by external KMS service. And this padding attack can carry out "without knowing the key"